crates/codegen/parser/runtime/src/kinds.rs

```diff
@@ -45,18 +45,17 @@ impl RuleKind {
 }
 
 /// The lexical context of the scanner.
-#[derive(strum_macros::FromRepr)]
-#[cfg_attr(feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "language"))]
-#[cfg_attr(not(feature = "slang_napi_interfaces"), derive(Clone, Copy))]
-pub enum LexicalContext {
+#[derive(strum_macros::FromRepr, Clone, Copy)]
+#[allow(clippy::upper_case_acronyms)] // An explicit placeholder replaced by the template engine
```

Comment:
nit: is it possible to use a correct case instead of the `#[allow]`?

Reply:
Replaced with a comment in https://github.com/NomicFoundation/slang/pull/728/commits/8f2001778ae6ec4859ccc9d56a00b75152105c85